system/lib/libc/dynlink.c

```diff
@@ -462,6 +494,16 @@ static struct dso* _dlopen(const char* file, int flags) {
 
   struct dso* p;
 
+  /* Resolve filename using LD_LIBRARY_PATH */
+  char buf[2*NAME_MAX+2];
```

Comment:
why this particular size?

Reply:
Just copied this from musl internal code.